src/Week.jsx

```diff
@@ -10,11 +10,7 @@ let Week = React.createClass({
   propTypes: TimeGrid.propTypes,
 
   getDefaultProps() {
-    return {
-      now: new Date(),
-      type: 'gutter',
-      timeslots: 2
-    }
+    return TimeGrid.defaultProps
```

Comment:
in case you didn't know, you could have done `TimeGrid.getDefaultProps()` I believe. switching to a class is fine too